src/gui/symbology/qgssymbollayerwidget.h

```diff
@@ -437,7 +437,9 @@ class GUI_EXPORT QgsSvgMarkerSymbolLayerWidget : public QgsSymbolLayerWidget, pr
   public slots:
     void setName( const QModelIndex &idx );
     void populateIcons( const QModelIndex &idx );
-    void setSize();
+    void setWidth();
```

Comment:
Can we make these private slots? I don't like how all these widgets expose a lot of their internal workings publicly (and hence become part of stable API). If not, they need docstrings added